R/eml_get.R

```diff
@@ -19,8 +21,8 @@
 #' @importFrom jqr jq combine
 #' @importFrom emld as_json as_emld
 #' @importFrom jsonlite fromJSON
-eml_get <- function(x, element, ...){
-  doc <- as.character(emld::as_json(emld::as_emld(x)))
+eml_get <- function(x, element, from = "xml", ...){
```

Comment:
@isteves I don't think we want to default to `xml` on `eml_get`.  I think `eml_get` would more typically be used after calling `read_eml` (as in the example), in which case the object is already a `list` (well, an `emld` S3 object, which is just a list with class attribute). 

Does the example still work when `from="xml"`?

```r
f <- system.file("xsd/test", "eml-datasetWithUnits.xml", package = "EML")
eml <- read_eml(f)
eml_get(eml, "physical")
```

Reply:
Yeah I thought so, too, but it seems to work fine, actually.  I even tried to test how long it takes, and it seems to be pretty similar.  Do you think it could slow down a larger EML file?

```
f <- system.file("xsd/test", "eml-datasetWithUnits.xml", package = "EML")
eml <- read_eml(f)
tic()
eml_get(eml, "physical")
toc()

tic()
eml_get(eml, "physical", from = "list")
toc()
```